src/web/components/App.tsx

```diff
@@ -629,24 +637,181 @@ export const App = ({ CAPI, NAV }: Props) => {
 					/>
 				</HydrateOnce>
 			))}
-			{CAPI.documentBlockElements.map((documentBlockElement, index) => (
+			{CAPI.documentBlockElements.map((element) => (
 				<HydrateOnce
-					key={index}
 					root="document-block-element"
-					index={documentBlockElement.documentIndex}
+					index={element.documentIndex}
 				>
 					<ClickToView
-						role={documentBlockElement.role}
-						isTracking={documentBlockElement.isThirdPartyTracking}
-						source={documentBlockElement.source}
-						sourceDomain={documentBlockElement.sourceDomain}
+						role={element.role}
+						isTracking={element.isThirdPartyTracking}
+						source={element.source}
+						sourceDomain={element.sourceDomain}
 						abTests={CAPI.config.abTests}
 					>
 						<DocumentBlockComponent
-							embedUrl={documentBlockElement.embedUrl}
-							height={documentBlockElement.height}
-							width={documentBlockElement.width}
-							title={documentBlockElement.title}
+							embedUrl={element.embedUrl}
+							height={element.height}
+							width={element.width}
+							title={element.title}
+						/>
+					</ClickToView>
+				</HydrateOnce>
+			))}
+			{CAPI.embedBlockElements.map((element) => (
+				<HydrateOnce
```

Comment:
Can't recall if this is a thng still (does React 17 fix this?) but we might need a `key` prop here (and elsewhere)

Reply:
Ah ok, interesting, I did wander what those ids were for.